### ISSUE TITLE
A11y_AzureToolsStorageExplorer_PromoteSnapshot_PromoteSnapshotPopup_ScreenReader: NVDA  is reading the dialogue title as row 1 column1 which is incorrect.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1890]
**Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

**GitHubTags:** #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS2.4.2; #NVDA; #Win10;

**Environment Details:**
Desktop App: Windows Version: Windows10
NVDA Version: 2019.1.1

**Steps to Reproduce:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob by using "Upload Files" control.
6. Navigate to "Create Snapshot" control and hit enter and navigate to "Manage snapshot" and hit enter.
7. Navigate to "Promote Snapshot" control and hit enter.
8. Verify that all the elements of "Promote Snapshot" screen are accessible and MAS Compliant.

**Actual:**
NVDA  is reading the dialogue title as row 1 column1 which is incorrect.

**Same issue is observed throughout the application for all the titles of dialogues present.**

**Same issue is observed in below Scenarios:**
1. **Toolbar Controls:**
* Here in this Scenario Screen reader unnecessary announces "rows and columns" for controls(Undelete Selected, Undelete by date) present under "Undelete" menu item.

**Expected:**
NVDA  is reading the dialogue title as row 1 column1 which is incorrect it should only read the dialogue title as "Microsoft Azure storage explorer".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

User Impact:
Screen reader user unable to know that which dialog is opened.

**MAS Reference:
MAS1.3.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

**MAS2.4.2:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={c74ff881-82b6-4401-92cf-8919f9bffaf2} 
Attachment for Reference
![1890_A11y_AzureToolsStorageExplorer_PromoteSnapshot_PromoteSnapshotPopup_ScreenReader](https://user-images.githubusercontent.com/55691739/65485311-2a38dc00-debf-11e9-998e-2e6d7dc231af.png)

**Comment by JasonYeMSFT** [comment]
Upon my investigation, electron is generating this problem since v2.0.0 and fixed it since v6.0.0. We need to upgrade electron to a higher version to resolve this issue.

**Comment by Alexandria-R** [comment]
Approved for exclusion as low impact.

**Comment by vinodbk197** [comment]
@MRayermannMSFT 
Verified this issue on: Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
As part of core scenario test execution, This issue is still reproduces for the below scenario.

1. Connect to Azure Storage
2. Delete Popup
3. Upload Files
4. Upload Folder
5. Update Access Tier
6. Clone with New Name Blob
7. Break Lease Popup
8. Rename Blob Container

**Comment by 12-shweta** [comment]
GitHubTags: #Win32;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
